reqKick.app.js

```diff
@@ -8,8 +8,18 @@ var worker = require('./worker.js');
 function setupGlobals() {
   global.who = 'reqKick|reqKick.app.js';
   global.logger = require('./common/logger.js')();
-  // TODO: Add an ENV for this.
-  logger.level = 'debug';
+
+  var runMode = process.env.RUN_MODE;
+  var logLevel;
```

Comment:
Lets make the default log level as warn, just in case we get an unknown runMode.